repository/src/main/java/tech/ydb/yoj/repository/db/Table.java

```diff
@@ -158,9 +160,7 @@ default <ID extends Entity.Id<T>> Stream<ID> readTableIds() {
         return readTableIds(ReadTableParams.getDefault());
     }
 
-    default FirstLevelCache getFirstLevelCache() {
-        return null;
-    }
+    FirstLevelCache<T> getFirstLevelCache();
```

Comment:
Suggestion:
I propose removing getFirstLevelCache from the Table interface entirely.

Reasoning:

In our project, this method isn’t used at all.

In YOJ code, it’s only needed in the default find method (where it can be obtained from the context (Tx.Current.get().getRepositoryTransaction()) instead).
This change would simplify the interface and reduce unnecessary coupling.

Reply:
:white_check_mark: Discussed and decided to postpone this to another PR. Apparently, not everyone uses `Tx`, some (at least, our tests) use the raw `RepositoryTransaction` (and then we'll need to expose `Table.getRepositoryTransaction()` which is even worse).